### PR TITLE
OOP exersice

### PR DESCRIPTION
**Pull request by iva-stasia**
### OOP exersice

[Demo](https://iva-stasia.github.io/a-tiny-JS-world-OOP-exercise/) |
[Code base](https://github.com/iva-stasia/a-tiny-JS-world-OOP-exercise)

The code is submitted in a dedicated feature branch.

Only code files are submitted.

Please, review.

### CODE REVIEW

**Comment by iva-stasia** [comment]
I checked it. It seems to me that everything is fine.

**Comment by iva-stasia** [comment]
I realized that I created Catwoman wrong. But I couldn't rewrite it using composition. I tried to find another solution, but when the cat's greeting changed, the Catwoman continued to say the old one. I don't seem to be able to find the right solution right now, so I'm deleting Catwoman for now.

**Comment by iva-stasia** [comment]
While I was working on another task, I came up with a Catwoman solution. I hope such an option, as I did, is possible.

File: submissions/iva-stasia/oop-exercise/index.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
+import { print } from './js/lib.js';
+
+class Inhabitant {
+   constructor(species, name, gender, legs, saying) {
+      this.species = species,
```
Comment (OleksiyRudenko) [inline submissions/iva-stasia/oop-exercise/index.js:5]: Why commas?

**Review by OleksiyRudenko — APPROVED** [review]
@iva-stasia well done!

File: submissions/iva-stasia/oop-exercise/index.js (hunk 1 above)
Comment (iva-stasia) [reply submissions/iva-stasia/oop-exercise/index.js:5]: I forgot that commas are not used in classes and was confused with the syntax of objects. Fixed in my repo.

**Pull request merged**